app/controllers/registration/early_years_experiences_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Registration
+  class EarlyYearsExperiencesController < BaseController
+    def edit; end
+
+    def update
+      form.early_years_experience = user_params[:early_years_experience]
+
+      if form.save
+        if current_user.registration_complete?
+          redirect_to user_path, notice: t(:details_updated)
+        else
+          redirect_to edit_registration_training_emails_path
+        end
+      else
+        render :edit, status: :unprocessable_entity
+      end
+    end
+
+  private
+
+    def user_params
+      params.require(:user).permit(:early_years_experience)
+    end
+
+    # @return [Registration::RoleTypeForm]
```

Comment:
class in the documentation needs updating here

Reply:
[0311c15](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/1048/commits/0311c150ffbddbc337cfa0da56eb08845edddd1b)